src/script/cpp_api/s_internal.h

```diff
@@ -57,7 +57,7 @@ class LockChecker {
 		SCRIPTAPI_LOCK_CHECK;                                                  \
 		realityCheck();                                                        \
 		lua_State *L = getStack();                                             \
-		assert(lua_checkstack(L, 20));                                         \
+		assert(lua_checkstack(L, 50));                                         \
```

Comment:
Why do we need this much elements in the stack? Also, perhaps note this in the commit message.